packages/shared/src/modules/users/libs/enums/notification-frequency.enum.ts

```diff
@@ -0,0 +1,6 @@
+const NotificationFrequency = {
+	ALL: "true",
+	NONE: "false",
```

Comment:
```suggestion
	ALL: "all",
	NONE: "none",
```
Let's leave a possibility to add other options in future